File: Programmers/LEVEL_1/12910-나누어_떨어지는_숫자_배열.js
/**
* ABOUT
*
* DATE: 2021-03-31
* AUTHOR: lovelysi0113
*
* URL: https://programmers.co.kr/learn/courses/30/lessons/12910
*
* COMMENT:
* filter() 메서드를 사용하여 divisor로 나누어 떨어지는 값들이 있는 배열을 구한다
**/

function solution(arr, divisor) {
    var answer = arr.filter(num => num % divisor === 0);
    answer.sort((a, b) => a-b);
    
    if (answer.length === 0) {
        return [-1];
    }
    return answer;
}

File: Programmers/LEVEL_1/12910-나누어_떨어지는_숫자_배열.py
'''/**
* ABOUT
*
* DATE: 2021-03-31
* AUTHOR: lovelysi0113
*
* URL: https://programmers.co.kr/learn/courses/30/lessons/12910
*
* COMMENT:
* divisor로 나누어 떨어지는 값들을 새 배열에 넣는다
**/'''

def solution(arr, divisor):
    answer = []
    for num in arr:
        if num % divisor == 0:
            answer.append(num)
    if not answer:
        return [-1]
    else:
        return sorted(answer)